### PR TITLE
Feature / Create & Style Get-start Section

### PR DESCRIPTION
**Pull request by liemgumball**
- Used import CSS files into index.css file
- Created Feature and Get-start section followed BEM naming convention
image
- Applied Flexbox
image
- Added hover animation for buttons and no action
- Added images into assets/images folder

### CODE REVIEW

File: practice-1/src/assets/css/get-start_section.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.get-start-section {
```
Comment (huyenpham-agilityio) [inline practice-1/src/assets/css/get-start_section.css:1]: file name not good

File: practice-1/src/assets/css/get-start_section.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    width: 750px;
+    height: 620px;
+}
+
+.get-start__image--earth {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 660px;
+    z-index: -99;
```
Comment (huyenpham-agilityio) [inline practice-1/src/assets/css/get-start_section.css:23]: not good

File: practice-1/src/assets/css/get-start_section.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
+.get-start-section {
+    position: relative;
+    top:auto;
```
Comment (huyenpham-agilityio) [inline practice-1/src/assets/css/get-start_section.css:3]: code style

File: practice-1/src/assets/css/get-start_section.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+.get-start__image--earth {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 660px;
+    z-index: -99;
+}
+
+.get-start__image--destinations {
```
Comment (huyenpham-agilityio) [inline practice-1/src/assets/css/get-start_section.css:26]: destinations is modifier in BEM?

File: practice-1/src/assets/css/get-start_section.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+#icon--add {
+    left: 400px;
+    bottom: -5px;
+}
+
+#icon--location {
+    bottom: 200px;
+    right: 0;
+}
```
Comment (huyenpham-agilityio) [inline practice-1/src/assets/css/get-start_section.css:53]: avoid style for id please

File: practice-1/src/assets/css/global.css [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+.text--sub {
+    color: var(--sub-color);
+}
+
+.text--small {
+    font-size: var(--font-size-small);
+}
+
+.text--smaller {
+    font-size: var(--font-size-smaller);
```
Comment (huyenpham-agilityio) [inline practice-1/src/assets/css/global.css:18]: use `smaller` is not good. Think about other name like tiny

File: practice-1/src/assets/css/index.css [hunk 7]
```diff
@@ -15,28 +17,48 @@ html {
   --font-color: #19182580 50% transparent;
   --sub-color: #19182580 ;
   --highlight-color: ##f85e9f;
-  --font-size-link: 15px;
+  --font-size-smaller: 15px;
   --font-size-small: 18px;
   --font-size-normal: 24px;
   --font-size-big: 35px;
   --font-size-bigger: 44px;
```
Comment (huyenpham-agilityio) [inline practice-1/src/assets/css/index.css:24]: bigger is not good too, large is a good option

File: practice-1/src/assets/css/index.css [hunk 8]
```diff
@@ -15,28 +17,48 @@ html {
...
   padding: 20px;
   margin: 0;
   transition: 0.25s ease;
 }
+
 h1, h2, h3 {
   font-family: 'Circular Std Medium', Arial, sans-serif;
+}
+
+#content {
```
Comment (huyenpham-agilityio) [inline practice-1/src/assets/css/index.css:51]: avoid style for id

File: practice-1/src/assets/images/OBJECTS1.svg
Comment (huyenpham-agilityio) [inline practice-1/src/assets/images/OBJECTS1.svg:1]: this is removed in design

File: practice-1/src/assets/images/Top destination.png
Comment (huyenpham-agilityio) [inline practice-1/src/assets/images/Top destination.png:1]: image name

File: practice-1/src/index.html [hunk 9]
```diff
@@ -3,8 +3,10 @@
   <head>
     <title>Travlog</title>
     <link rel="stylesheet" href="../src/assets/css/index.css" />
+    <link rel="stylesheet" href="../src/assets/css/global.css" />
     <link rel="stylesheet" href="../src/assets/css/header.css" />
     <link rel="stylesheet" href="../src/assets/css/footer.css" />
+    <link rel="stylesheet" href="../src/assets/css/get-start_section.css" />
```
Comment (huyenpham-agilityio) [inline practice-1/src/index.html:9]: import all css file in one file then add that file here

File: practice-1/src/assets/css/get-start_section.css (hunk 1 above)
Comment (liemgumball) [reply practice-1/src/assets/css/get-start_section.css:1]: how should I name this file?

Comment (liemgumball) [reply practice-1/src/assets/css/get-start_section.css:1]: I changed it into get-start.css

**Comment by liemgumball** [comment]
I have fix all the comments, please review it @huyenpham-agilityio:

- Added the reset.css and imported into index.css
- Renamed file into get-start.css
- Renamed variables

File: practice-1/src/assets/css/index.css [hunk 10]
```diff
@@ -1,42 +1,72 @@
+@import url(./reset.css);
+@import url(./global.css);
+@import url(./header.css);
+@import url(./footer.css);
+@import url(./get-start.css);
+
 html {
```
Comment (huyenpham-agilityio) [inline practice-1/src/assets/css/index.css:7]: move out all css to other file, this file just only for importing

File: practice-1/src/assets/images/Top_destination.png
Comment (huyenpham-agilityio) [inline practice-1/src/assets/images/Top_destination.png:1]: should be top-destination or topDestination

**Pull request merged**